### ISSUE TITLE
Is there way to ignore some commits from changelog

### ISSUE THREAD

**Issue by barmaglot92** [conventional-changelog/conventional-changelog#342]
Hello i want to remove some commits eg. "Merge in".. from changelog, how can i do this? I try to use headerPattern in parserOpts, but it doesn't help.

**Comment by matthew-bettingcloud** [comment]
I had a similar issue as yours, I'm using the gulp-conventional-changelog version, but should be pretty much similar. You can solve this by adding a pattern in grep, use both grep and inverse-grep combined. The pattern should match the commit you want excluded, be specific as much as possible.
I'm not sure this answers your question though... I'm new to conventional-changelog anyway :)

**Comment by gkalpak** [comment]
Being able to specify one or more patterns of commit messages to be ignored, would be really helpful. For example, there are certain scopes that we don't want to include in the changelog.

@matthew-bettingcloud, can you show an example of how exaclty you are using `grep` to achieve this?

**Comment by matthew-bettingcloud** [comment]
with use gulp-conventional-changelog, gitRawCommits will call git log with some specified parameters. You can add to those default parameters, simply just as the following...
```
return gulp.src("Changelog.md")
       .pipe(conventionalChangelog( {},{}, 
    {
     debug: x => console.log("[DEBUG]", x), <--- put this so that you can see the git log request and can try it manually.
      grep: "Merge", <--- this can be a simpe string or a pattern
      inverse-grep: true <---- this retreives anything not matched by the pattern.
    }));
```
don't forget that in grep by default it's basic regex, but you can switch to extended-regex. Also don't forget that you need to add a \ cause shell will remote it. Hope this helps.

**Comment by stevemao** [comment]
Yup, you could pass any git command to the module. Maybe we can add a recipe page. Feel free to contribute to that 😄

**Comment by blysik** [comment]
I run this via the conventional-changelog CLI tool, in a non-nodejs project.  I would like the commit message for autogenerating the changelog in our CI system to be ignored.  Can someone show me how to do this with just the standard config file?  (https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/index.js)

Thanks!
